ejercicio.md

```diff
@@ -0,0 +1,30 @@
+Sub recolecta()
+    donaron = 0
+    noDonaron = 0
+    totalRecaudo = 0
+    promedioRecaudo = 0
+    diezMil = 0
+    siguiente = "si"
+
+    While siguiente = "si" And totalRecaudo < 3000000
+        donar = Int(InputBox("Ingrese la cantidad a donar"))
+        totalRecaudo = donar + totalRecaudo
+        If donar >= 10000 Then
+            diezMil = diezMil + 1
+        End If
+        donaron = donaron + 1
+        siguiente = InputBox("Alguien más va a donar?")
```

Comment:
There could be conflicts here when executing this new InputBox. You already have one of these assigning above.